### DISCUSSION TITLE
Windows, server isn't included in the downloads?

### DISCUSSION THREAD

**Discussion by xKeiro** [Syncplay/syncplay#517] [Archive]
I wanted to run a Syncplay server, read the official support page about how to do it, but the windows zip package doesn't include the mentioned `syncplayServer.exe`. So how else could I run a server on windows?

**Comment by Et0h** [comment]
The server executable was removed from the bundle because it was resulting in Syncplay getting a false positive as per #458 and #456. While we did what we could to report it as a false positive, removing the executable from the release seemed like the quickest way to make the Syncplay client accessible again. Whether we go back to bundling it in future releases or end up just always having it separate to reduce false positives has yet to be decided.

The latest development build of Syncplay includes the server executable though, so just get it from Syncplay_1.7.0_Portable at https://github.com/Syncplay/syncplay/actions/runs/1934118179
